Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -284,15 +327,19 @@ namespace gdjs {
      * Reads the event sent by the authentication window and
      * display the appropriate banner.
      */
-    const receiveMessageFromAuthenticationWindow = function (
+    const receiveMessageFromLiluo = function (
       runtimeScene: gdjs.RuntimeScene,
       event: MessageEvent,
       { checkOrigin }: { checkOrigin: boolean }
     ) {
       const allowedOrigin = 'https://liluo.io';
 
       // Check origin of message.
-      if (checkOrigin && event.origin !== allowedOrigin) {
+      if (
+        checkOrigin &&
+        event.origin !== allowedOrigin &&
+        event.origin !== 'http://localhost:4000'
```

Comment:
Nit: you can do a `allowedOrigins` array and use includes to check if event.origin is allowed.

Also add gd.games for future proofing games?

Reply:
yeah this was just for test.
I'll add it!